vignettes/sensors.Rmd

```diff
@@ -16,138 +16,163 @@ knitr::opts_chunk$set(
 )
 ```
 
-`mhealthtools` offers a set of tools to extract features from some of the sensors present in almost all of the modern day phones. These include accelerometer and gyroscope data. We present examples to do so
+`mhealthtools` offers a set of tools to extract features from some of the sensors present in almost all of the modern day phones. Here we present examples from accelerometer and gyroscope data.
 
 Let's load the package first
-```{r}
+```{r mhealthtools}
 library(mhealthtools)
 ```
 
-```{r,echo=FALSE,message=FALSE,warning=FALSE, error=FALSE, prompt=FALSE}
+```{r other libraries, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, prompt=FALSE}
 library(dplyr)
 ```
 
 ## Accelerometer
 
 #### Data format and Sample data
 
-`mhealthtools` already comes preloaded with sample accelerometer data. You can access the data through `data(accelerometer_data)`. The first five rows are shown below, to give you an idea of the structure of the data (this is the input format to the functions that extract accelerometer features). Where `t` is the timestamp; `x`,`y` and `z` are the X,Y and Z- axis respectively
+`mhealthtools` already comes preloaded with sample accelerometer data. You can access the data through `data(accelerometer_data)` or just `accelerometer_data`. The first five rows are shown below, to give you an idea of the structure of the data (this is the input format to the functions that extract accelerometer features). Where `t` is the timestamp; `x`, `y` and `z` are the X, Y and Z- axis respectively.
 
-```{r data, echo = FALSE}
-data(accelerometer_data)
-head(accelerometer_data,5)
+```{r accel data, echo = FALSE}
+head(accelerometer_data, 5)
 ```
+*To use the default feature extraction pipelines you need to get the data in the proper format -- as that of the sample data. You can also define your own features or feed in a custom model to extract features in `mhealthtools` as described in [Extending mhealthtools](extending.html) under the "Sensor modules" section header.*
 
-Let's plot to see how the data actually looks like 
+Let's plot to see how the accelerometer data actually looks like 
 
-```{r, echo=FALSE, fig.width=8, fig.height=4}
+```{r accel plots, echo=FALSE, fig.width=8, fig.height=4}
 library(ggplot2)
 a <- accelerometer_data
 a$t <- a$t - a$t[1]
-a <- tidyr::gather(a, 'axis' , 'val', -t)
-ggplot(a, aes(x = t, y= val)) + geom_line()+facet_wrap(~axis) + ylim(c(-0.025, 0.025))
+a <- tidyr::gather(a, 'axis' , 'value', -t)
+ggplot(a, aes(x = t, y= value)) + geom_line() + 
+  facet_wrap(~axis, dir = 'v', strip.position = 'right') + ylim(c(-0.025, 0.025))
 ```
+We are looking at 10s of data sampled at a high enough sampling rate (for accelerometer data) of 100Hz. The data looks noisy, and looks like it could make good use of some pre-processing like frequency filtering, maybe subsetting the time (time filtering) -- all of which `mhealthtools` conveniently offers.
```

Comment:
10s of data sampled at 100Hz (High enough sampling rate for most use cases of accelerometer data)

Reply:
note to self: remove "high enough"